universal-application-tool-0.0.1/conf/messages

```diff
@@ -1,18 +1,29 @@
 # This is the default language for the application. Please provide context for each message for use in translations.
 # If a translation cannot be found for a given language, it will default to this file's text instead.
 
-# NO TRANSLATION NEEDED - this appears before applicants select their preferred language, so we always use the default.
-content.selectLanguage=Please select your preferred language from the following:
+# NOTES ON THIS FILE
+# 1. Apostrophes in translations must be escaped by another apostrophe. See https://www.playframework.com/documentation/2.8.x/JavaI18N#Notes-on-apostrophes
+# 2. Sections correspond to a single view, such as the applicant's home page or a view of an address question.
+# 3. We use the following key format: ${component_type}.${descriptive_name} - for example, button.nextPage would
```

Comment:
(non-blocking) can we add more examples here? e.g. a `button.ariaLabel.deleteEntity` and `placeholder.address.street`